requirements.txt

```diff
@@ -0,0 +1,2 @@
+mkdocs-material==9.1.6
+mkdocs-multirepo-plugin==0.6.1
```

Comment:
Please forgive me, but I just noticed that we are using poetry, ideally everything should be located inside the poetry configuration.

You can add a [new group of dependency with poetry](https://python-poetry.org/docs/managing-dependencies/) and setup those. You will then need to adapt your workflows to install those deps with poetry instead.